svd-rs/src/enumeratedvalue.rs

```diff
@@ -0,0 +1,130 @@
+use super::{BuildError, EmptyToNone, SvdError, ValidateLevel};
+
+/// Describes a single entry in the enumeration.
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+#[derive(Clone, Debug, PartialEq)]
+#[non_exhaustive]
+pub struct EnumeratedValue {
+    /// String describing the semantics of the value. Can be displayed instead of the value
+    pub name: String,
+
+    /// Extended string describing the value
+    #[cfg_attr(feature = "serde", serde(default))]
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
```

Comment:
this can be merged to either

```rust
#[cfg_attr(feature = "serde", serde(default, skip_serializing_if = "Option::is_none"))]
```

or

```rust
#[cfg_attr(feature = "serde", serde(default), serde(skip_serializing_if = "Option::is_none"))]
```

Reply:
Did not know about this. Thanks.